conf/messages.en

```diff
@@ -548,10 +528,7 @@ pensionSchemeInputAmounts.change.hidden = pension scheme {0} input amounts
 whichScheme.title = Which scheme are you entering details for?
 whichScheme.heading = Which scheme are you entering details for?
 whichScheme.hint = Select from the options below or enter a new scheme.
-whichScheme.New = Enter a new pension scheme
```

Comment:
Double check this please :)

Reply:
removed as not in use, bug raised to address the welsh translation